### DISCUSSION TITLE
轻松的沟通4 | /posts/soft-communication4/

### DISCUSSION THREAD

**Discussion by giscus** [earfanfan/yuanfan#619] [Announcements]
https://yuanfan.rbind.io/posts/soft-communication4/

**Comment by HushWay** [comment]
不知你有没有看过蔡智恒的小说《蝙蝠》，里面的短篇《米克》讲了狗狗米克与狗主一生相伴的故事，很感动。以后工作了，我也想养只狗
还有你们都好可爱，和小孩子玩真的很快乐欸

**Reply by earfanfan** [reply]
在你告诉我之前没有看过，不过你告诉我之后，我刚很有兴趣地搜了一下网上有，哈哈，晚上下班回去看看。如果你将来考虑养狗的话，一定要想清楚，要是决定好了就要对狗狗的一生负责哦。

和小孩子玩确实很快乐，而且沟通起来非常轻松，是那种心灵上的轻松。

**Comment by hongtaoh** [comment]
南方就是美啊。

**Reply by earfanfan** [reply]
你这说的，我忽然好奇想瞅瞅北方的村子长啥样了。

**Reply by hongtaoh** [reply]
一般都是灰灰的，植被没有南方茂盛，而且也没有那么多河流。

**Reply by earfanfan** [reply]
灰灰的啊，我们村冬天和初春也有灰灰的时候，不过现在也有，比如下面第一张黑黑的鸭子在灰灰的水泥地上依偎着睡觉，第二张是我前两周回去拍的，前一轮玉米被收割以后地里就会留下黄色的杆子，第三张是成都美术馆拍的，原图的颜色是那种看一眼就觉得幸福温暖的金黄色，不过拍下来变成偏灰的颜色了。

![2024-08-13-1](https://github.com/user-attachments/assets/c46839f3-849e-42ea-93ea-51e0634e55f9)
![2024-08-13-2](https://github.com/user-attachments/assets/ed89ea8d-90d3-46f6-857a-3fe8a8ca86f0)
![2024-08-13-3](https://github.com/user-attachments/assets/1fe1e295-725d-402f-9fe3-024e7d8de158)

**Reply by hongtaoh** [reply]
那这么说的话，感觉就是水有差别了。另外，你的第二张图在北方平原也可以看到。

**Reply by earfanfan** [reply]
郝糊涂啊，我想吐个槽，其实我在现实生活中也总碰到有人将一些差别简单地归因为性别差异，但其实仔细想想就会发现那也许只是个体差异。哈哈，就像你从没去我家乡的村子看过，居然会第一反应觉得是南北差异，那条河流是汉江支流，南方也有村子是不靠着大河的，北方应该也有村子是靠着大河的吧，毕竟北方也有江河。

**Reply by hongtaoh** [reply]
北方的河还是比南方少一些。

**Reply by earfanfan** [reply]
嗯，南方雨水丰沛，到了汛期，江河涨水就会变得很可怕，话说你自己印象中的南方北方分界线是哪咧？如果你把湖北当做南方的话，河南算是北方么？

**Reply by hongtaoh** [reply]
河南大部分算是北方，但是信阳那块就很靠近湖北了，就不好分了。河流应该对人的性格会有影响，我现在感觉，所以南北方有不同的性格。

**Reply by earfanfan** [reply]
我非常非常好奇你说的河流对人的性格影响如果展开讲讲是什么内容，说到南北方差异我最先想到的就是人们说话的口音不同，语速快慢不同，食物分量不同，如果非要说有撒性格差异的话，这里我瞎展开一下，可能南方宗族社会底蕴更浓厚，人们讲话更注重维持表面和谐，但是容易心里憋着生闷气不直接说……？

**Reply by hongtaoh** [reply]
这就是一个宏大的社会学议题了，超出了我的学术范围。我的直觉是在河边长大的人性格会更灵动和开放，没那么保守。

**Reply by earfanfan** [reply]
你的这个直觉带来的印象好特别，一般人都会觉得经济越发达的地区生活的人思想越开放反之更保守，相较来说，你这印象流真得好清新。
